shared/tests/utility/math/docs/spher_coords_cpp.txt

```diff
@@ -0,0 +1,256 @@
+Eigen::Matrix<double, 3, 1>(-5, 0, 0),
```

Comment:
I'd recommend removing these files as they clutter up the repo and don't add much. The scripts that made them are an important part. In general, you don't ever want to commit anything that was generated as it's fragile.